flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/options/JdbcConnectionOptions.java

```diff
@@ -32,15 +31,15 @@
 
 /** Options for the JDBC connector. */
 @Internal
-public class JdbcConnectorOptions extends JdbcConnectionOptions {
+public class JdbcConnectionOptions extends org.apache.flink.connector.jdbc.JdbcConnectionOptions {
```

Comment:
It would be better to change this class to another name, not same as the existing `JdbcConnectionOptions`

Reply:
Changed